src/coredump/coredump.c

```diff
@@ -826,7 +826,7 @@ static int submit_coredump(
         _cleanup_(sd_json_variant_unrefp) sd_json_variant *json_metadata = NULL;
         _cleanup_close_ int coredump_fd = -EBADF, coredump_node_fd = -EBADF;
         _cleanup_free_ char *filename = NULL, *coredump_data = NULL, *stacktrace = NULL;
-        const char *module_name, *root = MOUNT_TREE_ROOT;
+        const char *module_name, *root = NULL;
```

Comment:
Nah, please make commits clean...

Reply:
Don't focus on commits, I made them separate so they are easier to review, maybe that was a bad idea.  I will probably just squash them.